packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryNav.tsx

```diff
@@ -268,6 +264,28 @@ export const ChatbotConversationHistoryNav: FunctionComponent<ChatbotConversatio
             )}
           </DrawerActions>
         </DrawerHead>
+        <div className="pf-chatbot__input">
+          <Title headingLevel="h3">
+            <Icon size="lg">
+              <OutlinedClockIcon />
```

Comment:
I'm looking at our demo for "Drawer with simple menu" with these changes and I'm wondering if we might want to make this icon optional or customizable via props as well. This was not explicitly called out in the original ticket, but would you be willing to add this as well?